### PR TITLE
Allow URL to specify which test to display.

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
injecting Publish Params would be nice to have again too, can you add that to this PR... i think we can assume that a dataset must be displayed when doing so

File: demos/dermatology.html [hunk 1]
```diff
@@ -375,7 +379,7 @@
                 ;
                 if (first) {
```
Comment (mlzummo) [inline demos/dermatology.html]: when would this ever be false btw?

Comment (GordonSmith) [inline demos/dermatology.html]: This is inside the resize method - and the last line of code calls ``` frame.doResize();``` so on the "first call" we want to process the url, but never again.

**Comment by GordonSmith** [comment]
The publish param injection never went away:
http://rawgit.com/GordonSmith/Visualization/DERM_TEST_SEL/demos/dermatology.html?src/chart/Line&orientation=vertical

**Comment by mlzummo** [comment]
ahh i see, GH minimized the area of code where that's being done and i thought it was missing

**Comment by GordonSmith** [comment]
@mlzummo Good to pull?

**Comment by mlzummo** [comment]
yes

**Pull request merged**